src/ddmd/dimport.d

```diff
@@ -304,8 +304,13 @@ extern (C++) final class Import : Dsymbol
             {
                 AliasDeclaration ad = aliasdecls[i];
                 //printf("\tImport %s alias %s = %s, scope = %p\n", toPrettyChars(), aliases[i].toChars(), names[i].toChars(), ad._scope);
-                if (mod.search(loc, names[i]))
+
+                Dsymbol importedSymbol = mod.search(loc, names[i]);
+                if (importedSymbol)
                 {
+                    // This deprecation should turn into an error as soon as possible
+                    if (importedSymbol.prot().kind == PROTprivate)
+                        mod.deprecation(loc, "member '%s' is private", names[i].toChars());
```

Comment:
How does "module $module member $name is not accessible because it is private". I think having the word "deprecated" in the message is redundant since the phrase starts with "Deprecation: ".